docs/modules/publisher.md

```diff
@@ -121,24 +126,205 @@ A basic configuration file looks like this:
 ```
 
 OPC UA optimizes network bandwidth by only sending data changes to OPC Publisher when the data has changed. If data changes need to be published more often or at regular intervals, OPC Publisher supports a "heartbeat" for every configured data item that can be enabled by additionally specifying the HeartbeatInterval key in the data item's configuration. The interval is specified in seconds:
-```
+
+``` json
  "HeartbeatInterval": 3600,
 ```
 
 An OPC UA always send the current value of a data item when OPC Publisher first connects to it. To prevent publishing this data to IoT Hub, the SkipFirst key can be additionally specified in the data item's configuration:
-```
+
+``` json
  "SkipFirst": true,
 ```
 
 ### Configuration via Command Line Arguments
 
 There are several command line arguments that can be used to set global settings for OPC Publisher. They are described [here](publisher-commandline.md).
 
+### Configuration via IoT Hub Direct methods
+
+**Please note: This feature right now is only available starting with version 2.8.2.**
+
+OPC Publisher implements the following [IoT Hub Direct Methods](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-direct-methods) which can be called from an application (from anywhere in the world) leveraging the [IoT Hub Device SDK](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-sdks):
+
+Consists in the following methods that are inherited from 2.5.x plus some additional ones
+
+- PublishNodes_V1
+- UnpublishNodes_V1
+- UnpublishAllNodes_V1
+- GetConfiguredEndpoints_V1
+- GetConfiguredNodesOnEndpoints_V1
+- GetDiagnosticInfo_V1
+- AddOrUpdateNodes_v1
```

Comment:
Please rename `AddOrUpdateNodes_v1` to `AddOrUpdateEndpoints_V1`. We had a discussion about this during one of the review sessions and as far as I remember we decided that `AddOrUpdateEndpoints` will be a more descriptive name as we are changing endpoints here. I have just updated the spec with this new name.

Reply:
right